rpi_backlight.py

```diff
@@ -144,9 +176,23 @@ def _create_argument_parser() -> argparse.ArgumentParser:
     )
     return parser
 
+def init() -> None:
+    global MODE, PATH
+    try:
+        with open("/sys/firmware/devicetree/base/model") as f:
+            model_information = f.read()
+        if "Raspberry Pi" in model_information:
+            PATH = "/sys/class/backlight/rpi_backlight/"
+            MODE = MODE_RPI
+        elif "Tinker Board" in model_information:
+            PATH = "/sys/devices/platform/ff150000.i2c/i2c-3/3-0045/"
+            MODE = MODE_TINKERBOARD
+    except ValueError as error:
```

Comment:
That shouldn't be necessary... am I missing something? Where would a `ValueError` possibly occur?

Reply:
Would it be OSError?
I wasnt able to determine what kind of error it should be raised as. I said value because incorrect value in the file

Reply:
tried to fix. please review

Reply:
Is it possible that you're mixing up *catching an exception* and *raising an exception* here? We want the latter, and a `ValueError` is appropriate - but we don't need to *catch* an exception, because there should not be one.

So this should be good, I think you had this previously.

```python
with open("/sys/firmware/devicetree/base/model") as f:
    model_information = f.read()

if "Raspberry Pi" in model_information:
    PATH = "/sys/class/backlight/rpi_backlight/"
    MODE = MODE_RPI
elif "Tinker Board" in model_information:
    PATH = "/sys/devices/platform/ff150000.i2c/i2c-3/3-0045/"
    MODE = MODE_TINKERBOARD
else:
    raise ValueError("unsupported OS, or OS could not be detected!")
```